utility/mm2_update-master-directory-list

```diff
@@ -754,13 +651,14 @@ def setup_logging(config, options):
     else:
         log_file = options.logfile
 
-    fmt = '%(asctime)s %(message)s'
-    datefmt = '%m/%d/%Y %I:%M:%S %p'
-    formatter = logging.Formatter(fmt=fmt, datefmt=datefmt)
+    fmt = '%(asctime)s:%(category)s:%(message)s'
+    formatter = logging.Formatter(fmt=fmt)
```

Comment:
Yes the date formatter is dropped. Instead of defining a common date formatter the default (like in the crawler is used). That way both tools have the same format.